qt_ui/widgets/QTopPanel.py

```diff
@@ -127,15 +138,21 @@ def setGame(self, game: Optional[Game]):
         self.budgetBox.setGame(game)
         self.factionsInfos.setGame(game)
 
-        self.passTurnButton.setEnabled(True)
-        if game and game.turn > 0:
-            self.passTurnButton.setText("Pass Turn")
+        state = game.check_win_loss()
```

Comment:
Needs a comment explaining why this would happen. Are you trying to re-disable things if someone loads an already completed game? Or is this one of those `setGame` calls that actually happens every turn?

OTOH, if you take my suggestion to invert the ownership, this confusing bit probably just disappears.